### PR TITLE
[OWL-1124] Icon svg component

### PR DESCRIPTION
**Pull request by rwu823**
- Removed unused icons
- Added new script: `build.svg` and `postinstall`
- Rename to match new icons
- Added doc and testing

image

@chuanxd @CApopsicle

### CODE REVIEW

**Comment by chuanxd** [comment]
傳一份到 ftp 上備份吧，小心不要覆蓋到現有的 svg

**Comment by rwu823** [comment]
現在的版本，已經有 commit 上去了阿

File: icons/icon-add.svg [hunk 1]
```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
-    <!-- Generator: Sketch 3.5.2 (25235) - http://www.bohemiancoding.com/sketch -->
-    <title>icon/24/icon-add</title>
-    <desc>Created with Sketch.</desc>
-    <defs></defs>
-    <g id="out" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
-        <path d="M19.6785714,13.5357143 L13.5357143,13.5357143 L13.5357143,19.6785714 L10.4642857,19.6785714 L10.4642857,13.5357143 L4.32142857,13.5357143 L4.32142857,10.4642857 L10.4642857,10.4642857 L10.4642857,4.32142857 L13.5357143,4.32142857 L13.5357143,10.4642857 L19.6785714,10.4642857 L19.6785714,13.5357143 L19.6785714,13.5357143 L19.6785714,13.5357143 Z" id="path" fill="#000000" sketch:type="MSShapeGroup"></path>
-    </g>
-</svg>
```
Comment (chuanxd) [inline icons/icon-add.svg:10]: Why removed ?

**Review by chuanxd — CHANGES_REQUESTED** [review]

File: icons/icon-add.svg (hunk 1 above)
Comment (rwu823) [reply icons/icon-add.svg:10]: duplicate  with `icon-plus`

image

**Pull request closed without merging**